src/ninja.cc

```diff
@@ -915,7 +940,7 @@ const Tool* ChooseTool(const string& tool_name) {
     printf("ninja subtools:\n");
     for (const Tool* tool = &kTools[0]; tool->name; ++tool) {
       if (tool->desc)
-        printf("%10s  %s\n", tool->name, tool->desc);
+        printf("%11s  %s\n", tool->name, tool->desc);
```

Comment:
Could you elaborate on this change?

Reply:
This is to align the list of tools properly -- the "missingdeps" line stands out otherwise with its short description. That's because "missingdeps" happens to be longer than the other tool names.
Roughly:
```
     browse  browse dependency graph in a web browser
      clean  clean built files
   commands  list all commands required to rebuild given targets
       deps  show dependencies stored in the deps log
missingdeps  check deps log dependencies on generated files
```
vs
```
    browse  browse dependency graph in a web browser
     clean  clean built files
  commands  list all commands required to rebuild given targets
      deps  show dependencies stored in the deps log
missingdeps  check deps log dependencies on generated files
```

Reply:
Sounds good.